virtualcontext.js

```diff
@@ -8,14 +8,13 @@
 const Types = require("./types");
 const Context = require("./context");
 
-function VirtualContext(id, endpoint=null, parent=null)
+function VirtualContext(id, src=null, parent=null)
```

Comment:
Wouldn't it be more consistent to rename `src` to `virtualSrc`?

Reply:
done!